modules/ROOT/pages/cloudhub-architecture.adoc

```diff
@@ -4,13 +4,20 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: cloudhub, cloud, manage, mmc, api, worker, runtime manager, arm
 
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
+
+
 xref:cloudhub.adoc[CloudHub] has been built from the ground up to provide enterprises with a multi-tenant, secure, elastic, and highly available integration platform as a service (iPaaS). This document describes how the underlying mechanisms of the CloudHub platform work to achieve these goals.
```

Comment:
Well, since you mentioned the white...:
```suggestion
xref:cloudhub.adoc[CloudHub] is designed to provide enterprises with a multitenant, secure, elastic, and highly available integration platform as a service (iPaaS). To maximize your use of CloudHub, It is important to understand how the underlying mechanisms of the CloudHub platform work to achieve these goals.
```